components/gantt/dependencies/overview.md

```diff
@@ -21,9 +21,10 @@ The Telerik Gantt for Blazor allows you define dependencies, which are rendered
 
 ## Gantt Dependencies Features:
 
-* Data Binding - You can provide a collection of dependencies to the Gantt Chart for Blazor. For more information read the [Data Binding]({%slug gantt-dependencies-databind%}) article.
-
-* `TypeField` - `string` - Defines the dependency type. For more information read the [Types]({%slug gantt-dependencies-types%}) article. 
-
-* `Editing` - You can allow the user edit the dependencies. For more information read the [Editing]({%slug gantt-dependencies-editing%}) article.
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
 
+| Feature | Description |
+| --- | --- |
+| `Data Binding` | Provide a collection of dependencies to the Gantt Chart for Blazor. For more information read the [Data Binding]({%slug gantt-dependencies-databind%}) article. |
```

Comment:
We are mixing features and parameters here. I had to stare for a few seconds to understand what is `Data Binding` (in a code block). Consider different structure or presentation.